logger/logger_test.go

```diff
@@ -90,22 +90,6 @@ func TestLogConfigsetLoggerToFile(t *testing.T) {
 	}
 }
 
-func TestLogConfigsetLoggerToStdout(t *testing.T) {
-
-	confFile := &Config{
-		Output: "Filename",
-	}
-	confStdout := &Config{
-		Output: "Stdout",
-	}
-
-	confStdout.setLoggerToStdout()
-	assert.Equal(t, os.Stdout, log.StandardLogger().Out)
-	confFile.setLoggerToStdout()
-	assert.Equal(t, os.Stdout, log.StandardLogger().Out)
-
-}
-
 func TestConfigsetLoggerToGraylog(t *testing.T) {
```

Comment:
So bug is found, tests are deleted but no one new test introduced to check this bug. There is a good practice  to create a test for the bug found, so please add a couple.

Reply:
test added